src/components/errorPage/actions/primary/BackToLogin.tsx

```diff
@@ -6,9 +6,10 @@ import { ActionButtonInterface } from '../interface';
 
 const BackToLogin: FC<ActionButtonInterface> = ({ t, language = 'de' }) => {
   const languageToUse = language === 'en' ? 'de' : language;
-  const logoutLink = `/${languageToUse}/account/logoff`;
-  const loginLink = `/${languageToUse}/account/logon`;
-  const query = `?returnurl=${encodeURIComponent(loginLink)}`;
+  const logoutLink = `/seller-web/api/auth/logout`;
+  const vehicleManagementLink = `/${languageToUse}/vehicle-management`;
+  const loginLink = `/seller-web/api/auth/login?locale=${languageToUse}&returnTo=${encodeURIComponent(vehicleManagementLink)}`;
```

Comment:
i might be missing context. Shouldn't this be configurable? Not sure we always want to go to vehicle pool?

Reply:
So it's a bit tricky. Ideally yes and we would also pass in the proxy segments. But the page is hosted on seller web so we need to stay within seller web to make it work properly. Otherwise we would need one auth error page per project. So for simplicity I added VP for now so the user don't end up not logged in on the HP after re-login